plugins/lsp/language-servers/default.nix

```diff
@@ -310,6 +310,16 @@ let
       name = "metals";
       description = "metals for Scala";
     }
+    {
+      name = "nextls";
+      description = "The language server for Elixir that just works.";
+      package = pkgs.next-ls;
+      serverName = "nextls";
+      cmd = cfg: [
+        "${cfg.package}/bin/nextls"
```

Comment:
Could you use `(lib.getExe cfg.package)` here please?

Reply:
Done